tools/JSMaker.js

```diff
@@ -30,8 +30,8 @@ exports.init = function() {
 
 
 exports.visitPOM = function(pom) {
+  if ( X.version ) version = X.version;
```

Comment:
This should go in the init() method instead so you only do it once, not once for each POM.

Reply:
moved.